src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -86,7 +89,8 @@ describe('<Dropdown />', () => {
     const renderMethod = () => {
       render(dropdownWithChildrenAndButtonText);
     };
-
+    // expect console error from react, suppressed.
+    jest.spyOn(console, 'error').mockImplementation();
```

Comment:
nit: you can place this in `beforeEach` to apply to each test. also it would be good to add a `afterEach` to reset, so that we don't leave any mocks between tests

https://jestjs.io/docs/jest-object#jestspyonobject-methodname-accesstype

Reply:
I didn't place in a `beforeEach` in case an error actually needs to be thrown during testing. However, I did add `mockRestore`s after each use for clean up